XenOvfTransport/Export.cs

```diff
@@ -329,6 +329,11 @@ private EnvelopeType _export(Session xenSession, string targetPath, string ovfna
                 {
                     OVF.AddOtherSystemSettingData(ovfEnv, vsId, "hardware_platform_version", vm.hardware_platform_version.ToString(), OVF.GetContentMessage("OTHER_SYSTEM_SETTING_DESCRIPTION_1"));
                 }
+
+                if (vm.HasSriovRecommendation())
+                {
+                    OVF.AddOtherSystemSettingData(ovfEnv, vsId, "allow_network_sriov", Messages.TRUE, OVF.GetContentMessage("OTHER_SYSTEM_SETTING_DESCRIPTION_1"));

```

Comment:
I don't think we should use a resource message here, as these messages are localised. 
We can use bool.TrueString or bool.TrueString.ToLowerInvariant() if we want the lowercase "true"